### ISSUE TITLE
Go icon almost disappears, depending on the UI Theme.

### ISSUE THREAD

**Issue by uchoa** [DanBrooker/file-icons#216]
Maybe that's due to too light lines. I don't really know. But here's what happens:

<img width="188" alt="go-icon" src="https://cloud.githubusercontent.com/assets/587124/10619317/bd8fae16-7753-11e5-91d3-ef9df808a522.png">

In this example the theme is visual-studio-code.

**Comment by uchoa** [comment]
If I knew how to fix it, I would.

**Comment by Alhadis** [comment]
Yikes, that's bad. Does adding the following chunk to your theme's CSS help?
```css
.icon-file-text[data-name$=".go"]:before{
	-webkit-font-smoothing: subpixel-antialiased;
	-webkit-text-stroke-color: transparent !important;
	font-weight: bolder !important;
}
```
I use a light-coloured theme so I can't replicate too easily... *but* it seems like there's a faint black outline being applied to the icon (which isn't present on the other icons):
![1](https://cloud.githubusercontent.com/assets/2346707/10636840/038fde36-784e-11e5-8522-ce498b41bf8c.png)

**Comment by Alhadis** [comment]
[Fixed in v1.7.0](https://github.com/DanBrooker/file-icons/blob/master/CHANGELOG.md#170---2016-04-02).
